### ISSUE TITLE
Bump the new version

### ISSUE THREAD

**Issue by thunderbirdtr** [psifidotos/applet-window-buttons#128]
Hello, I'm working on to make fedora RPM package of your applet and prepare to push into fedora repository but It would be nice to have bump to version with your recent changes, so I can avoid the create patch for recent plasma version in order to make it work. in version 0.8.1 without patch, it won't going to work. Since this is a breaking change, on latest plasma It would be nice to have it as a upstream release tag. 

Example :  https://build.opensuse.org/package/show/KDE:Extra/applet-window-buttons
Patch : https://build.opensuse.org/package/view_file/KDE:Extra/applet-window-buttons/fix-build-plasma-5.21.patch?expand=1

Thank you :)

**Comment by psifidotos** [comment]
Yes you are right, I will try to find time to release this as version 0.9.0
